### PR TITLE
Updated readme with more information

### PR DESCRIPTION
**Pull request by TSunny007**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
 There are the known limitations with the private preview
 
-- Lock/unlock resources is unavailable. Lock/unlock performs an operation on the set of resources to prevent (Lock) or enable (Unlock) changes.
-- What-if is unavailable. What-if allows for evaluating changes before deploying.
-- The purge mode of a stack does not purge resourceGroups, subscriptionAliases, or managementGroups that are created by the stack.
-- Can't create managed group level stacks.
-- It is not recommended to use stack in production environment because this release returns secured strings, secured objects, and the template.
-- Bicep support not available.
-- Portal support not available.
+- It is not recommended to use deploymentStacks in production environment since a deploymentStack doesn not obey secured strings or objects.
```
Comment (bmoore-msft) [inline README.md:14]: What does it mean to say "secureStrings or objects" aren't obeyed?

Also we should separate the "don't use in prod" and the "secure param" bullet points

File: README.md [hunk 2]
```diff
@@ -397,9 +401,9 @@ After updating the stack, use `Get-AzSubscriptionDeploymentStack` to list the re
 
 ## Use snapshots
 
-Snapshots provide a way to view the history of updates to the stack. The resource type is read-only. New snapshots are created by virtue of updating the parent stack. Snapshots are primarily used for viewing history for diagnostics or troubleshooting. In the future, you can roll back a stack to a previous snapshot if there is an error.
+Snapshots provide a way to view the history of updates to the stack. A snapshot is read-only and is created whenever any template resource is successfully deployed.  Note that not all resources need to be successfully deployed for a snapshot to be created. Snapshots are primarily used for viewing history for diagnostics or troubleshooting. In the future, a snapshot can be applied to roll back a deploymentStack to a previous state if there is an error.
```
Comment (bmoore-msft) [inline README.md:404]: I made an edit - can you confirm this is accurate:

```A snapshot is read-only and is created whenever any template resource is successfully deployed. Note that not all resources need to be successfully deployed for a snapshot to be created.```

**Review by bmoore-msft — COMMENTED** [review]
I made some edits and left some comments...

File: README.md (hunk 2 above)
Comment (TarunSunkaraneni) [reply README.md:404]: Noteworthy distinction, good add

File: README.md (hunk 1 above)
Comment (TarunSunkaraneni) [reply README.md]: Done, i don't think if fully understand securestring and secureobjects, but have made a clarifying change

**Review by bmoore-msft — COMMENTED** [review]
LGTM

**Pull request merged**